src/main/java/com/github/akosbordas/ncore/TorrentDetails.java

```diff
@@ -35,6 +36,7 @@ public TorrentDetails parse(String html) {
         Document document = parseHtml(html);
 
         try {
+            english = extractGenericInfoColumn(1, 2, document).contains("EN");
```

Comment:
Ez a megoldas nem teljesen megbizhato igy es nem is altalanos. Vannak olyan torrent tipusok ahol ennek a mezonek nincs ertelme. Egy lehetsegesen jo altalanos megoldas lehetne, ha egy ```TorrentType``` peldanyt tarolnank el egy mezoben. Ez mar rendelkezni fog azzal a tulajdonsaggal, hogy angol-e a torrent vagy sem es altalanos is, hiszen tipusa minden torrentnek van. 

A tipus linkjebol kinyerheto fenti informacio:
```html
<div class="dd"><a title="Még több ebből a kategóriából" href="torrents.php?csoport_listazas=osszes_jatek">Játék</a> > <a title="Még több ebből a kategóriából" href="torrents.php?tipus=game_rip">PC/RIP</a></div>
```

Reply:
Mi lenne, ha a parseType() függvény egy TorrentTypeot adna vissza, és azt eltárolnánk a TorrentDetails-ben? (ha jól eredetileg is TorrentTypeot adott vissza, csak amikor átjavítottam osztályra, az egyszerűség kedvéért átírtam ezt a függvényt String-re)